### DISCUSSION TITLE
Typical Chemical Descriptor Values

### DISCUSSION THREAD

**Discussion by Allen2019** [rdkit/rdkit#7361] [FAQ]
In the RDKit documentation for the chemical descriptors, one might see an entry like this:

> rdkit.Chem.MolSurf.SMR_VSA8(x, y=7)
>     MOE MR VSA Descriptor 8 (3.63 <= x < 3.80)

(from [this page](https://rdkit.org/docs/source/rdkit.Chem.EState.EState_VSA.html#rdkit.Chem.EState.EState_VSA.EState_VSA1))

Is the range enclosed in parenthesis on the second line the range one should expect from most molecules? Should one be concerned if the value is outside of that?

For example, the SMR_VSA8 value is zero for all of the molecules from a project I'm working on. (And I decided that this probably isn't a mistake on my end because this value is also zero in this [tutorial](https://greglandrum.github.io/rdkit-blog/posts/2022-12-23-descriptor-tutorial.html) from the RDkit blog).

So, do those numbers indicate the typical range? If not, what do they mean?

**Comment by bp-kelley** [comment]
@Allen2019 for the VSA descriptors, you are in luck, Greg wrote a blog on it last year:

https://greglandrum.github.io/rdkit-blog/posts/2023-04-17-what-are-the-vsa-descriptors.html
